app/api/v2/handlers/ability_api.py

```diff
@@ -22,35 +22,47 @@ def add_routes(self, app: web.Application):
         router.add_patch('/abilities/{ability_id}', self.update_ability)
         router.add_delete('/abilities/{ability_id}', self.delete_ability)
 
-    @aiohttp_apispec.docs(tags=['abilities'])
+    @aiohttp_apispec.docs(tags=['abilities'], summary='Get all abilities.',
+                          descriptions='Provides a list of all available abilities.')
     @aiohttp_apispec.querystring_schema(BaseGetAllQuerySchema)
     @aiohttp_apispec.response_schema(AbilitySchema(many=True, partial=True))
```

Comment:
For each endpoint, can you provide a `description` for each `response_schema`? For this it can just be something like "Returns a list of all abilities". The others can be some variation of "Json dictionary representation of the created/updated/replaced Agent", and the delete request will just return HTTP 204 Status Code (No Content)

Reply:
Sorry forgot about that part 😅